tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -91,16 +92,19 @@ def test_reverting_container(
                 zero_section,
                 revert_sub_container,
             ],
+            kind=ContainerKind.INITCODE
```

Comment:
the container kind matches the `deploy_tx`, so we can set the expected kind in the EOFStateTest.

Reply:
We can do: if `deploy_tx` is unspecified, and either `container_kind` or `container.kind` are set to `ContainerKind.INITCODE` then assume `deploy_tx==True`.